### PR TITLE
add direction to rotate3d

### PR DESCRIPTION
**Pull request by nden**

### CODE REVIEW

**Comment by nden** [comment]
Jut a note that this isn't ready to merge yet.

**Comment by nden** [comment]
@mdboom I added the standard euler rotation sequences here. Let me know if you prefer them in a separate schema.

**Comment by mdboom** [comment]
I don't think it needs to be a separate schema -- but maybe add a short description of what each value means?

**Comment by nden** [comment]
Done. I think not all transform schemas are showing in the built documentation, not sure why. For example the projections are missing.

File: schemas/stsci.edu/asdf/0.1.0/transform/rotate3d.yaml [hunk 1]
```diff
@@ -6,7 +6,16 @@ tag: "tag:stsci.edu:asdf/0.1.0/transform/rotate3d"
...
+  Euler angle rotation around 3 axes.
+
+  `direction` : Sequence of rotation axes: one of `zxz`, `zyz`, `yzy`, `yxy`, `xyx`, `xzx`
+                or `native2celestial`, `celestial2native`
+
+  `phi`, `theta`, `psi`: - Euler angles in degrees.
+                If `direction` is one of `native2celestial` or `celestial2native`,
+                `phi`, `theta` are the longitude and latitude of the native pole in
+                the celestial system and `psi` is the longitude of the celestial pole in
+                the native system.
```
Comment (mdboom) [inline schemas/stsci.edu/asdf/0.1.0/transform/rotate3d.yaml]: These descriptions should probably go with the properties themselves (below).

**Comment by mdboom** [comment]
Just tried building your branch, and all of the transformation schemas (including the projections) do seem to be included.

**Comment by nden** [comment]
I don't see them (or at least expected to see more) here

http://asdf-standard.readthedocs.org/en/latest/schemas/transform.html#projections

**Comment by mdboom** [comment]
Ah -- that seems to be something readthedocs specific.  I'll see if the logs say anything.

**Comment by mdboom** [comment]
I've fixed readthedocs.

**Comment by nden** [comment]
I've nothing to add here.

**Pull request merged**